client/hotrod-client/src/test/java/org/infinispan/client/hotrod/AuthenticationTest.java

```diff
@@ -40,7 +40,7 @@ protected EmbeddedCacheManager createCacheManager() throws Exception {
       return cacheManager;
    }
 
-   private ConfigurationBuilder initServerAndClient() {
+   protected ConfigurationBuilder initServerAndClient() {
```

Comment:
This method doesn't seem to be used in new tests - do we need to change it to protected?